deployments/docker/app/config/lbrytv.yml

```diff
@@ -15,3 +15,6 @@ MetricsAddress: :2112
 MetricsPath: /metrics
 
 PublishSourceDir: /storage/publish
+BlobFilesDir: /storage/lbrynet/blobfiles
+
+Reflector: reflector.lbry.com:5566
```

Comment:
I would call this ReflectorURL

Reply:
That might be misleading as it's technically not a URL

Reply:
You can call it `ReflectorHostPort` if you want to be very precise

Reply:
Renamed to `ReflectorAddress`.